tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +79,34 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+#     raise Exception("Complete this test according to comments below.")
+#     # *********************************************************************
+#     # ****** Complete Assert Portion of this test **********
+#     # *********************************************************************
+#     # Assertions should check:
+#     # - That the results is truthy
+#     # - That tai and jesse's inventories are the correct length
+#     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    
+    # Assert
+    assert result == True
+    expected_length_tai = len(tai.inventory)
+    assert expected_length_tai == actual_length_tai
+
+    expected_length_jesse = len(jesse.inventory)
+    assert expected_length_jesse == actual_length_jesse
+
+    assert (item_f in tai.inventory) == True
+    assert (item_c not in tai.inventory) == True 
+    assert (item_c in jesse.inventory) == True
+    assert (item_f not in jesse.inventory) == True 
```

Comment:
We don't need the `== True` here. `assert`s automatically succeed when given a truthy value and automatically fail when given a falsey value.